contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,294 @@
+
+So PostHog and Statsig are similarly priced for A/B testing and feature flags, although PostHog is more expensive for product analytics.
+
+![Statsig pricing](../images/blog/posthog-vs-statsig/posthog-price.png)
+
+#### Example scenarios
+
+To give you an idea of what pricing looks like in reality, here are some example situations and their estimated costs for both PostHog and Statsig. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+       <tr>
+            <td><strong>A/B testing and feature flag requests</strong></td>
+            <td><strong>PostHog cost</strong></td>
+            <td><strong>Statsig cost</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+       <tr>
+            <td>**1,000,000**</td>
+            <td>$0</td>
+            <td>$0</td>
+        </tr>
+       <tr>
+            <td>**5,000,000**</td>
+            <td>$240</td>
+            <td>$150</td>
+        </tr>
+       <tr>
+            <td>**10,000,000**</td>
+            <td>$460</td>
+            <td>$400</td>
+        </tr>
+       <tr>
+            <td>20,000,000</td>
+            <td>$720</td>
+            <td>$900</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> **Notes:** You can significantly lower your number of feature flag and A/B testing requests by using [PostHog's backend local evaluation](/docs/feature-flags/local-evaluation), potentially saving you a good amount of money.
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+       <tr>
+            <td><strong>Product analytics events</strong></td>
+            <td><strong>PostHog cost</strong></td>
+            <td><strong>Statsig cost</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+       <tr>
+            <td>1,000,000</td>
+            <td>$0</td>
+            <td>$0</td>
+        </tr>
+       <tr>
+            <td>5,000,000</td>
+            <td>$700</td>
+            <td>$150</td>
+        </tr>
+       <tr>
+            <td>10,000,000</td>
+            <td>$1,350</td>
+            <td>$400</td>
+        </tr>
+       <tr>
+            <td>20,000,000</td>
+            <td>$2,400</td>
+            <td>$900</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+### Integrations
+
+Both PostHog and Statsig have a range of integrations that enable them to import, export, enhance, and make use of data. However, a key difference is that since PostHog is open source, it's possible to [create your own integration](/docs/apps/build).
+
+Below is a sample comparison of PostHog and Statsig's integrations. Be sure to checkout [PostHog's full list of integrations](/cdp).
+
+<ComparisonTable column1="PostHog" column2="Statsig">
+   <ComparisonRow column1={true} column2={true} feature="Imports" description="Import data from source" />
+   <ComparisonRow column1={true} column2={true} feature="Exports" description="Export data to other sources" />
```

Comment:
These are both enterprise features, should we mention that?

Reply:
`Data Warehouse Imports` is an enterprise feature, but `Incoming Data Integrations`  and `Outgoing Data Integrations` are available on their Pro plan